### PR TITLE
Favourite icon

### PR DESCRIPTION
**Pull request by owennw**
Adds the favourite icon to the favourite tiles.

### CODE REVIEW

**Comment by bjedrzejewski** [comment]
@owennw will review this now

File: src/sidebars/star.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    'use strict';
+
+    angular.module('openfin.star', ['openfin.store'])
+        .controller('StarCtrl', ['$scope', 'storeService', function($scope, storeService) {
+            var self = this;
+
+            var favouriteColours = {
+                on: '#42D8BD',
+                off: '#1A1F26',
+                offhover: "#263337"
```
Comment (bjedrzejewski) [inline src/sidebars/star.js]: Shall we make it to css classes already? Colours in JS are bad.

File: src/sidebars/favourites/favourites.js [hunk 2]
```diff
@@ -2,117 +2,126 @@
...
-                            }
-                        });
-                    }
-                })
-            };
+                self.update();
 
-            self.updateFavourites();
-        }])
+                $scope.$on('favouriteChanged', function(event, data) {
```
Comment (bjedrzejewski) [inline src/sidebars/favourites/favourites.js]: Do we have to use the dreaded $scope?

Comment (owennw) [inline src/sidebars/favourites/favourites.js]: When the icon is clicked we need to
a) remove the local storage entry
b) remove the favourite from the search bar
c) remove the favourite from the favourites

The only ways I can think of doing all of these are to
a) call several functions from the html ng-click on different (unrelated?) controllers
b) emit an event and catch it in the relevant places.

I opted for the event as I felt it was a valid use of it, and leads to tidier code -- I hope! And to do that we need to use `$scope`.

Can you think of another way to either use an event, or not use the event at all?

File: src/sidebars/star.js (hunk 1 above)
Comment (owennw) [inline src/sidebars/star.js]: Yes this should be improved I think. These colours are all already defined in our .less file anyway. Thankfully this refactor has brought all of these colours into one file. Should this be done as part of this PR or as a later tidy? I've already created an issue (#139) to tidy around the less file.

File: src/sidebars/favourites/favourites.js (hunk 2 above)
Comment (bjedrzejewski) [inline src/sidebars/favourites/favourites.js]: It actually is more cross cutting than i thought. I think event is the right way to go.

**Pull request merged**